### PR TITLE
Fructose/android ci

### PR DESCRIPTION
**Pull request by kaliabadi**

### CODE REVIEW

File: packages/brightcove-video/brightcove-player.fructose.android.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/* globals withComponent test beforeEach driver */
+import React from "react";
+import BrightcovePlayer from "./brightcove-player";
+import VideoWithExternalControls from "./fixtures/video-with-external-controls";
+
+const policyKey =
```
Comment (L0wry) [inline packages/brightcove-video/brightcove-player.fructose.android.js]: are these okay being here?

File: packages/brightcove-video/brightcove-player.fructose.android.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  <BrightcovePlayer
+    fructoseID="brightcove video renders"
+    policyKey={policyKey}
+    videoId={videoId}
+    accountId={accountId}
+  />,
+  "video with external controls",
+  async fructose => {
+    beforeEach(async () => {
+      await fructose.loadComponent();
```
Comment (L0wry) [inline packages/brightcove-video/brightcove-player.fructose.android.js]: we're not resetting the app here like in the other tests, is this on purpose?

Comment (kaliabadi) [reply packages/brightcove-video/brightcove-player.fructose.android.js]: Addressed

File: packages/brightcove-video/brightcove-player.fructose.android.js (hunk 1 above)
Comment (kaliabadi) [reply packages/brightcove-video/brightcove-player.fructose.android.js]: Yeah checked with Jack and he said it’s ok

**Pull request merged**